backend/knexfile.js

```diff
@@ -2,6 +2,7 @@ require("dotenv").config();
 const { loadSecretFileOrUseEnv } = require('./src/helpers/secrets');
 
 module.exports = {
+  supportBigNumbers: false,
```

Comment:
What does this do?

Reply:
I don't know.

Reply:
https://stackoverflow.com/questions/50327557/what-is-the-best-way-to-handle-big-number-returning-from-database-in-nodejs#50342463